app/src/main/res/values/strings.xml

```diff
@@ -433,4 +433,7 @@ Upload your first media by touching the camera or gallery icon above.</string>
   <string name="display_location_permission_title">Display location permission</string>
   <string name="display_location_permission_explanation">Ask for location permission when needed for nearby notification card view feature.</string>
 
+  <string name="no_image">No images used</string>
+  <string name="no_image_reverted">No image reverted</string>
+  <string name="no_image_uploaded">No images Uploaded</string>
 </resources>
```

Comment:
I am not an English pro but "image" should be "images" and "Uploaded" -> "uploaded"

Reply:
Yes you are right here. Things must be at perfection level. I'll do that